### DISCUSSION TITLE
Add an enable attributes in PlanarControls

### DISCUSSION THREAD

**Discussion by valentinMachado** [iTowns/itowns#2045] [Ideas]
The PlanarControls implement an enableRotation, enableSmartTravel and enablePan attributes to enable or not certain aspect of the controls. It would be nice to have a global enable which enable or not the controls. This would allow to switch between controls without disposing creating a new PlanarControls everytime.

If this sounds good to you, I could propose a PR implementing this.

**Comment by jailln** [comment]
Sounds good to me :+1:
I think we should implement it for the `GlobeControls` too though to keep the two view's controls harmonized. Do you think you can do it for both?

**Reply by valentinMachado** [reply]
:+1: yes I can, it will be more or less the same thing

**Reply by valentinMachado** [reply]
I've done it for the PlanarControls which is quite inspire from [THREE.OrbitControls](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/controls/OrbitControls.js). I was about doing it in GlobeControls I found this [line](https://github.com/iTowns/itowns/blob/9b6d59f19ec5ee3e65b6a0f731365d51e5644060/src/Controls/GlobeControls.js#L195), try it in an example and it seems to work :smile: 

PR => https://github.com/iTowns/itowns/pull/2046

**Reply by jailln** [reply]
Ahah ok thanks :)
